spacecmd/src/spacecmd/softwarechannel.py

```diff
@@ -780,56 +780,65 @@ def do_softwarechannel_update(self, args):
     (args, options) = parse_command_arguments(args, arg_parser)
 
     if is_interactive(options):
-       options.label = prompt_user('Channel Label:', noblank=True)
-
-       print('')
-       print('New Name (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.name = prompt_user('Name:')
-
-       print('')
-       print('New Summary (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.summary = prompt_user('Summary:')
-
-       print('')
-       print('New Description (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.description = prompt_user('Description:')
-
-       print('')
-       print('New Checksum type (blank to keep unchanged)')
-       print('------------')
-       print('\n'.join(sorted(self.CHECKSUM)))
-       print('')
-       options.checksum = prompt_user('Select:')
-
-       print('')
-       print('New GPG URL (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.gpg_url = prompt_user('GPG URL:')
-
-       print('')
-       print('New GPG ID (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.gpg_id = prompt_user('GPG ID:')
-
-       print('')
-       print('New GPG Fingerprint (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.gpg_fingerprint = prompt_user('GPG Fingerprint:')
-
-       print('')
-       print('Disable GPG CHECK (blank to keep unchanged)')
-       print('------------')
-       print('')
-       options.disable_gpg_check = prompt_user('Disable GPG Check [yes/no]? ')
+
+        options.label = prompt_user('Channel Label:', noblank=True)
+        options.vendor_channel = is_vendor_channel(self, options.label)
+
+        if options.vendor_channel:
+           print('')
+           print('Disable GPG CHECK (blank to keep unchanged)')
+           print('------------')
+           print('')
+           options.disable_gpg_check = prompt_user('Disable GPG Check [yes/no]? ')
+        else:
```

Comment:
I would turn this arround and ask for `if not vendor_channel` .

Below there is again the "Disable GPG Check" question. This can be asked outside of the "if" in both cases.